Fix jwt docs since gateway v2 and k8s 1.16 (#579)
solo-io/solo-docs: gloo/docs/gloo_routing/virtual_services/security/jwt/access_control.md

Wait for the deployments to finish:
```shell
kubectl -n gloo-system rollout status deployment/discovery
kubectl -n gloo-system rollout status deployment/gateway-v2
kubectl -n gloo-system rollout status deployment/gloo
kubectl -n gloo-system rollout status deployment/gateway-proxy-v2
```

> Install the petstore demo app and add a route and test that everything so far works (you may need to wait a minutes until all the gloo containers are initialized):

Reviewer: ```suggestion
Install the petstore demo app, add a route, and test that everything so far works (you may need to wait a few moments for all the Gloo containers to be initialized):
```

>   bottom box (labeled "Private Key").

Reviewer: "We need have everything we need to sign and verify a custom JWT with our custom claims": this line is off as well.

You should now have an encoded JWT token in the "Encoded" box. Copy it and save to to a file called 
`token.jwt`

> {{% notice note %}}

Reviewer: 👏 👏 👏
